bibimainapp/src/main/java/de/unibi/cebitec/bibiserv/web/rest/BiBiServRESTServlet.java

```diff
@@ -4,15 +4,16 @@
  */
 package de.unibi.cebitec.bibiserv.web.rest;
 
-import com.sun.jersey.spi.container.servlet.ServletContainer;
 import de.unibi.cebitec.bibiserv.server.manager.Core;
+import org.glassfish.jersey.servlet.ServletContainer;
```

Comment:
The BiBiServRestServelet is used by BiBiServ tools which are not part of the bibimainapp. Updating Jersey to version 2.1 leads to a not working REST API for attached/deployed tools.